### PR TITLE
Nazar/feature/refactoring v2

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/pages/Error/Error.module.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
     align-items: center;
     justify-content: center;
     margin: 0 auto;
-    color: red;
+    color: #62aef6;
 
     font-size: 40px;
     font-weight: 800;
```
Comment (kipod) [inline frontend/src/pages/Error/Error.module.css:13]: px

File: frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfoUpdate.tsx [hunk 2]
```diff
@@ -45,18 +49,27 @@ const BusinessInfoUpdate = ({
...
+  } = useForm<Inputs>({
     defaultValues: {
       businessLogo: logo,
       businessName: name,
-      userEmail: user_email,
+      businessPhoneNumber: phoneNumberAutoFormat(phone_number),
     },
   });
 
+  console.log(errors, "errors");
```
Comment (kipod) [inline frontend/src/pages/Home/components/SettingsView/components/LeftPanel/BusinessInfoUpdate.tsx:65]: console.error

**Review by kipod — COMMENTED** [review]
One BUG found in the backend code

File: frontend/src/pages/AddProduct/components/SetProductInfo/SetProductInfo.tsx [hunk 3]
```diff
@@ -30,7 +33,15 @@ const SetProductInfo = ({ productDispatch }: SetProductInfoProps) => {
 
     if (payloadKey === "image") {
       if (!e.target.files) return;
-      payloadValue = e.target.files[0];
+
+      const file = e.target.files[0];
+      const isImage = validateImageFile(file);
+
+      if (!isImage) return;
```
Comment (Taraslysak) [inline frontend/src/pages/AddProduct/components/SetProductInfo/SetProductInfo.tsx:40]: I'd recommend display some tost to inform user about failed upload

File: frontend/src/pages/Error/Error.module.css (hunk 1 above)
Comment (Taraslysak) [reply frontend/src/pages/Error/Error.module.css:13]: Font weight has no units

**Review by Taraslysak — COMMENTED** [review]
Nice

**Pull request merged**